app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php

```diff
@@ -56,7 +56,7 @@ public function present(GetPlaceCollectionResponse $placeResponse): array
             $result['features'] = $this->featurePresenter->presentCollection($place->features);
             $result['localization'] = $this->localizationPresenter->presentCollection($place->localization);
             $result['category'] = $this->categoryPresenter->present($place->category);
-            $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->category->tags);
+            $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->tags);
```

Comment:
$place->tags loaded in repository with(...)?

Reply:
Done.